src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Do not disable eslint on props instead ensure that all props are properly validated